src/main/resources/public/js/getAndAddContracts.js

```diff
@@ -20,165 +20,147 @@ var projectContractsCount = -1;
 
     function getTasksOfContract(contract) {
         $("#tasksTable").dataTable().fnDestroy();
-        $("#tasksTable").find("tbody").html('');
         $("#tasksTitle").html(
             " Tasks assigned to " + contract.id.contributorUsername
             + " (" + contract.id.role + ")"
         )
         $("#tasks").show();
-        $("#loadingTasks").show();
-        $.ajax( //API call to get the Tasks.
-            "/api/projects/"
-            + contract.id.repoFullName
-            + "/contracts/" + contract.id.contributorUsername + "/tasks?role=" + contract.id.role,
-            {
-                type: "GET",
-                statusCode: {
-                    200: function (tasks) {
-                        $("#loadingTasks").hide();
-                        tasks.forEach(
-                            function(task) {
-                                $("#tasksTable").find("tbody").append(
-                                    taskAsTableRow(contract, task)
-                                );
-                            }
-                        );
-                        $("#tasksTable").dataTable();
-                        $("#tasksBody").show();
-                        console.log(tasks);
-                    },
-                    204: function (data) {
-                        $("#loadingTasks").hide();
-                        $("#loadingTasksHidden").show();
-                        $("#tasksTable").dataTable();
-                        $("#tasksBody").show();
-                    },
-
+        $("#tasksTable").DataTable({
+            language: {
+                loadingRecords: '<img src="/images/loading.svg" height="100">'
+            },
+            ajax: {
+                url: "/api/projects/"
+                    + contract.id.repoFullName
+                    + "/contracts/" + contract.id.contributorUsername + "/tasks?role=" + contract.id.role,
+                dataSrc: ""
+            },
+            columns: [
+                {
+                    data: "issueId",
+                    render: function (data) {
+                        var issueLink;
+                        if (contract.id.provider == 'github') {
+                            issueLink = 'https://github.com/'
+                                + contract.id.repoFullName
+                                + "/issues/"
+                                + data;
+                        } else if (contract.id.provider == 'gitlab') {
+                            issueLink = 'https://gitlab.com/'
+                                + contract.id.repoFullName
+                                + "/issues/"
+                                + data;
+                        } else {
+                            issueLink = '#';
+                        }
+                        return "<a href='" + issueLink + "' target='_blank'>#" + data + "</a>"
```

Comment:
@amihaiemil I think most of this formatting can be done on server, on json creation.
`task` json might look like
```javascript
{
  idTask: { id: '#1', link: 'http://github...'},
  assignmentDate: <formatted date>,
  deadline: <formatted date>,
  estimation: <formattted estimation>,
  value: <euro formatted value>
}
```
This way we only need to render in table the `a` element for the link.